### ISSUE TITLE
Simplify usage of environments

### ISSUE THREAD

**Issue by bittner** [thulasi-ram/django-feature-toggle#8]
We deploy _django-feature-toggles_ with our Django application to our target environments (development, integration, production).

With this approach, clearly,

- _production_ is only relevant on Production,
- _integration_ only on Integration,
- _development_ only on the Development target environment,

and we initialize the toggles via a management command at startup. (It's a cloud-native setup on a Kubernetes / OpenShift cluster.)

In other words, we don't need to specify the "environment" at all in our setup as we're not controlling multiple environment but only the one we're running on.

## Suggested Change

If this is a valid use case for you we should maybe make "environments" optional – and hidden on the Admin – by default. Only when specified either in the Django settings or populated in the app's model we may want to show it as a control.
